DevoxxClientMobile/src/main/java/com/devoxx/views/cell/ScheduleCell.java

```diff
@@ -51,13 +52,23 @@
 import javafx.scene.layout.StackPane;
 import javafx.scene.layout.VBox;
 
+import java.util.Arrays;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 import java.util.Optional;
 
 public class ScheduleCell extends CharmListCell<Session> {
 
     private static final PseudoClass PSEUDO_CLASS_COLORED = PseudoClass.getPseudoClass("color");
-
+    private static Map<String, String> trackStyleClassMap = new HashMap<>();
```

Comment:
To avoid performance issues, consider using pseudo classes.
`Map<String, PseudoClass> trackStyleClassMap`